babel.config.js

```diff
@@ -1,19 +1,9 @@
 module.exports = api => {
-  api.cache(false)
+  api.cache(true)
   return {
-    babelrcRoots: ['.', './packages/*', './products/*', './plugins/*'],
-    comments: true,
     presets: [
       '@babel/preset-react',
-      [
```

Comment:
reduced amount of env config, but note we cannot fully remove @babel/env as it appears to convert esm imports/exports to make it compatible with jest